include/zenoh-pico/transport/transport.h

```diff
@@ -81,34 +89,23 @@ typedef enum _z_unicast_peer_flow_state_e {
 
 typedef struct {
     _z_sys_net_socket_t _socket;
-    _z_id_t _remote_zid;
     // SN numbers
     _z_zint_t _sn_rx_reliable;
     _z_zint_t _sn_rx_best_effort;
-    volatile bool _received;
     bool _pending;
     uint8_t flow_state;
     uint16_t flow_curr_size;
     _z_zbuf_t flow_buff;
+    _z_transport_peer_common_t common;
```

Comment:
Is there any value in having the common member first to ensure a similar memory layout between structs containing the same information?